autoload/lsp/utils.vim

```diff
@@ -319,3 +319,11 @@ function! lsp#utils#make_valid_word(str) abort
    endif
    return l:str
 endfunction
+
+function! lsp#utils#_split_by_eol(text) abort
+    let l:text = a:text
+    let l:text = substitute(l:text, "\r\n", "\n", 'g')
+    let l:text = substitute(l:text, "\r", "\n", 'g')
+    return split(l:text, "\n", v:true)
```

Comment:
Equivarent to this?

```
return split(a.text, '\r\n\|\r', 1)
```

Reply:
Thanks for your review.
test case can pass with `split(a:text, '\r\n\|\r\|\n', v:true)`.

I will change to it.